contracts/LRTDepositPool.sol

```diff
@@ -237,7 +204,7 @@ contract LRTDepositPool is ILRTDepositPool, LRTConfigRoleChecker, PausableUpgrad
     /// @dev only callable by LRT admin
     /// @param nodeDelegatorAddress NodeDelegator contract address
     function removeNodeDelegatorContractFromQueue(address nodeDelegatorAddress) public onlyLRTAdmin {
```

Comment:
This is a bug in their original code, but `isNodeDelegator` needs to be cleared when an NDC is removed. (this should be right before the event emit.)

```
isNodeDelegator[nodeDelegatorAddress] = 0;
```

Reply:
Nice spot. I've added extra asserts to the unit tests to cover this